sparse/__init__.py

```diff
@@ -140,6 +140,8 @@
     nansum,
     result_type,
     roll,
+    sort,
+    take,
```

Comment:
This should be in `docs/generated/sparse.rst`

Reply:
Done!